modules/aerodyn/src/AeroDyn_Inflow.f90

```diff
@@ -392,6 +392,11 @@ subroutine ADI_InitInflowWind(Root, i_IW, u_AD, o_AD, IW, dt, InitOutData, errSt
       ! the average values for the entire wind profile must be calculated and stored (we don't know if OLAF
       ! is used until after AD_Init below).
       InitInData%BoxExceedAllow = .true.
+      
+      !bjj: what about these initialization inputs?
```

Comment:
Good question.  I don't know offhand.  Thanks for adding the comment -- hopefully we get around to addressing it a some point.